### DISCUSSION TITLE
Is it necessary to monitor and alarm for tracing?

### DISCUSSION THREAD

**Discussion by runtu666** [open-telemetry/community#1586] [Community Discussions]
Is it necessary to monitor and alarm for tracing, and if so, how should it be done? Which indicators should be alerted? Or is it just a means to troubleshoot problems, and check the tracing after there are error logs and performance monitoring alarms (Prometheus)

**Comment by trask** [comment]
hi @runtu666! you might have better luck asking in the [#opentelemetry](https://cloud-native.slack.com/archives/CJFCJHG4Q) slack channel
